pullrequest.rst

```diff
@@ -258,21 +258,30 @@ and for each pull request there may be several commits.  In particular:
 
 Commit messages should follow the following structure::
 
-   bpo-42: the spam module is now more spammy (GH-NNNN)
+   bpo-42: make the spam module more spammy (GH-NNNN)
 
    The spam module sporadically came up short on spam. This change
    raises the amount of spam in the module by making it more spammy.
 
 The first line or sentence is meant to be a dense, to-the-point explanation
-of what the purpose of the commit is.  If this is not enough detail for a
-commit, a new paragraph(s) can be added to explain in proper depth what has
-happened (detail should be good enough that a core developer reading the
-commit message understands the justification for the change).
+of what the purpose of the commit is. The imperative form is strongly
```

Comment:
Since we have many developers with varying levels of English proficiency, some of them may not be overly familiar with the difference between imperative and descriptive form. As a result, I think it's worth having `(used in the example above)` after "imperative form" to make it a bit more explicit as to what is being referred to.

E.g. "The imperative form (used in the example above) is strongly preferred..."